deptofdefense/move.mil
# PPM Estimator

Implements #102.

## Checklist

I have…


## Summary of Changes

- Adds a page where users can estimate the incentive the government would pay them if they chose to move their HHG on their own (a Personal Procurement Method move).

## Testing

To verify the changes proposed in this pull request…

1. clone this repo,
1. git checkout `ppm-estimator`,
1. set up development dependencies according to [CONTRIBUTING.md](https://github.com/deptofdefense/move.mil/blob/master/CONTRIBUTING.md),
1. run `bin/rails server`, and
1. load up [http://localhost:3000](http://localhost:3000) in your Web browser of choice.

## Screenshots

<details><summary>Desktop</summary>

![fireshot capture 60 - personally procured move ppm_ - http___localhost_3000_resources_ppm-estimator](https://user-images.githubusercontent.com/26554826/33785471-2ff983dc-dc33-11e7-893a-5b4272751cb1.png)

</details>

<details><Summary>Mobile</summary>

![fireshot capture 61 - personally procured move ppm_ - http___localhost_3000_resources_ppm-estimator](https://user-images.githubusercontent.com/26554826/33785487-3c18202e-dc33-11e7-924d-d9d93ab420da.png)

</details>
